LocalSettings.php

```diff
@@ -2975,6 +2975,71 @@
 			'amc' => true,
 		],
 	],
+	'wgMFQueryPropModules' => [
+		'default' => [ 
+			'pageprops', 
+		],
+		'gratisdatawiki' => [ 
+			'entityterms',
+		],
+	],
+	'wgMFSearchAPIParams' => [
+		'default' => [
+			'ppprop' => 'displaytitle',
+		],
+		'gratisdatawiki' => [
+			'wbetterms' => 'label',
+		],
+	],
+	'wgMFSearchGenerator' => [
+		'default' => [
+			'name' => 'prefixsearch',
+			'prefix' => 'ps',
+		],
+		'gratisdatawiki' => [
+			'name' => 'wbsearch',
+			'prefix' => 'wbs',
+		],
+	],
+	'wgMFVaryOnUA' => [
+		'default' => false,
+		'gratispaideiawiki' => true,
+	],
+	'wgMFUseWikibase' => [
```

Comment:
It seems like this was actually removed years ago (2018), but never from extension.json or any documentation? Seems to have been removed in https://github.com/wikimedia/mediawiki-extensions-MobileFrontend/commit/304c0b0923a263386c54f2a484b8a10a26cb094c.

Reply:
> It seems like this was actually removed years ago (2018), but never from extension.json or any documentation? Seems to have been removed in [wikimedia/mediawiki-extensions-MobileFrontend@304c0b0](https://github.com/wikimedia/mediawiki-extensions-MobileFrontend/commit/304c0b0923a263386c54f2a484b8a10a26cb094c).

No, this wasn't removed at all. A few was removed which I can't mention now cuz I'm mobile but not this.

Reply:
Usage of this variable literally does not exist in the extension anymore.

Reply:
> Usage of this variable literally does not exist in the extension anymore.

Okay, remove it then.